includes/class-simple-local-avatars.php

```diff
@@ -1222,7 +1233,44 @@ private function clear_user_avatar_cache( $local_avatars, $user_id, $media_id )
 	}
 
 	/**
-	 * Migrate the user's avatar data from WP User Avatar/ProfilePress
+	 * Add default avatar upload file field.
+	 *
+	 * @param array $defaults Default options for avatar.
+	 *
+	 * @return array Default options of avatar.
+	 */
+	public function add_avatar_default_field( $defaults ) {
+		if ( ! did_action( 'wp_enqueue_media' ) ) {
+			wp_enqueue_media();
+		}
+		$default_avatar_file_id  = get_option( 'simple_local_avatar_default', '' );
+		$default_avatar_file_url = wp_get_attachment_image_url( $default_avatar_file_id );
```

Comment:
We should check if `$default_avatar_file_id` is not empty. Also, `$default_avatar_file_url` should not be `false` before proceeding.